flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/history/HistoryServer.java

```diff
@@ -163,6 +175,8 @@ public HistoryServer(Configuration config, CountDownLatch numArchivedJobs) throw
 		}
 		webDir = new File(webDirectory);
 
+		boolean processArchiveDeletion = config.getBoolean(HistoryServerOptions.HISTORY_SERVER_PROCESS_JOB_DELETION);
```

Comment:
I think `cleanExpiredJobs` or `removeExpiredJobs` better describes this variable.

Reply:
Renamed, thanks